chapter07/mdw/Fred.txt

```diff
@@ -0,0 +1,18 @@
+Mr. Slate had an employee
+Fred Filntstone was his name
+fred was happy
+fred was married to wilma
+wilma and betty were friends
+fred was the father of bammbamm
```

Comment:
Wait, what? Wow. That adds whole new layers to the show. Did Wilma know? Did Barney? Wow. :scream: